app/models/wakes/location.rb

```diff
@@ -3,17 +3,24 @@ class Wakes::Location < ActiveRecord::Base
   include Wakes::Metrics::GoogleAnalyticsPageviews
   include Wakes::Metrics::Facebook
 
-  validates :path, :format => { :with => %r{\A\/} }, :uniqueness => true
+  validates :path, :format => { :with => %r{\A\/} }, :uniqueness => { :scope => :host }
   belongs_to :resource, :foreign_key => :wakes_resource_id, :inverse_of => :locations, :touch => true
 
   scope :canonical, -> { where(:canonical => true) }
   scope :legacy, -> { where(:canonical => false) }
 
   def label
-    path
+    (host + path if host.present?) ||
```

Comment:
@benhutton Could do. I was keeping it consistent with the abbreviated pseudo-URI syntax found elsewhere, namely `hostname/path`. This is not technically a URI (and the URI methods in Ruby do not parse it correctly). 

The shorter syntax helps with readability in [redirect_mapper_spec.rb](https://github.com/desiringgod/wakes/blob/01a02b9efa94278f5bbdc13da1615c7848b14ab2/spec/lib/wakes/redirect_mapper_spec.rb#L182-L232), and I thought the same considerations would apply when printing out the wakes graph. Do you still prefer the full URL?